src/Microsoft.OpenApi.OData.Reader/Edm/EdmModelExtensions.cs

```diff
@@ -197,5 +197,28 @@ public static bool IsOperationImportOverload(this IEdmModel model, IEdmOperation
             return model.EntityContainer.OperationImports()
                 .Where(o => o.Operation.IsBound == operationImport.Operation.IsBound && o.Name == operationImport.Name).Count() > 1;
         }
+
+        /// <summary>
+        /// Get all of the elements in the model and its referenced models.
+        /// </summary>
+        /// <returns>All the elements.</returns>
+        public static IEnumerable<IEdmSchemaElement> GetAllElements(this IEdmModel model)
```

Comment:
>GetAllElements(this IEdmModel model) [](start = 53, length = 36)

We should consider the size of the output.
The output of Open API only for Microsoft.Graph CSDL is very big.

Reply:
I mean that is the whole point of this PR - to allow you to make a big OAS file that is combined from multiple smaller CSDL files.   If it is too big, then it is too big, but most OAS tools don't support file composition either.